temoa/model_checking/unit_checking/relations_checker.py

```
@@ -0,0 +1,516 @@
+                io_units = tech_lut.get(tech_or_comm)
+                if not io_units:
+                    grouped_errors[
+                        f'Unprocessed row (missing reference for tech '
+                        f'"{tech_or_comm}" --see earlier tests)'
+                    ].append(idx)
+                    continue
+                expected_units = io_units.output_units
+            case RelationType.COMMODITY:
+                expected_units = comm_lut.get(tech_or_comm)
+            case _:
+                raise ValueError(f'Unexpected relation type: {relation_type}')
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)